src/Microsoft.Health.Fhir.Shared.Core.UnitTests/Features/Search/SearchParameters/SearchParameterDefinitionManagerTests.cs

```diff
@@ -0,0 +1,109 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Threading.Tasks;
+using MediatR;
+using Microsoft.Health.Fhir.Core.Features.Definition;
+using Microsoft.Health.Fhir.Core.Features.Search.Parameters;
+using Microsoft.Health.Fhir.Core.Features.Search.Registry;
+using Microsoft.Health.Fhir.Core.Models;
+using Microsoft.Health.Fhir.ValueSets;
+using NSubstitute;
+using Xunit;
+
+namespace Microsoft.Health.Fhir.Core.UnitTests.Features.Search
+{
+    public class SearchParameterDefinitionManagerTests
```

Comment:
Is the idea for this class to have tests for all the search parameter definition managers? Right now I think it makes more sense to name it for the SupportedSearchParameterDefinitionManager unless the other classes will have tests added to this file.

Reply:
I think the file could be used for more SearchParameterDefinitionManager tests as it has the infrastructure in place.  I added one more test that was different to make it slight more broad.